tool/ruby_vm/views/_insn_entry.erb

```diff
@@ -33,12 +37,29 @@ INSN_ENTRY(<%= insn.name %>)
 % if insn.handles_sp?
     POPN(INSN_ATTR(popn));
 % end
+#ifdef _FORTIFY_SOURCE
```

Comment:
How is this macro expected to be defined? `cppflags="-D_FORTIFY_SOURCE"` on configure? I think it would be nice if there were CI that enables this. I understand ko1 is running CI server that has `-DVM_CHECK_MODE=1` (or 2?), so I think defining `_FORTIFY_SOURCE` depending on `VM_CHECK_MODE` is an option.

Reply:
I don't think `_FORTIFY_SOURCE` is good macro to enable checking because this macro seems system provided checking. Or my misunderstanding?
Now we have `VM_CHECK_MODE` to enable VM related checking and ci.rvm.jp runs a test with enabling this macro.

Reply:
OK.  Will use VM_CHECK_MODE instead.